lib/openscap_parser/rule.rb

```diff
@@ -35,10 +39,15 @@ def references
 
     def identifier
       @identifier ||= {
-        label: @rule_xml.at_css('ident')&.text,
+        label: @rule_xml.at_css('ident') && @rule_xml.at_css('ident').text,
```

Comment:
Why this change? You don't like the new `&.` syntax?

Reply:
Perhaps the proxy has to run on an older version of ruby...?

Reply:
Yeah - I'm not blocking this PR on it, just curious.

Reply:
Yeah - I'm not blocking this PR on it, just curious.

Reply:
Exactly, proxies may have older versions of ruby.